packages/@lightningjs/ui-components/src/components/Button/ButtonSmall.d.ts

```diff
@@ -17,9 +17,27 @@
  */
 
 import lng from '@lightningjs/core';
-import Button from './Button';
+import Button, { ButtonStyle } from './Button';
+import { StylePartial } from '../../types/lui';
+import type { TextBoxStyle } from '../TextBox';
 
-export default class ButtonSmall<
-  TemplateSpec extends Button.TemplateSpec,
-  TypeConfig extends lng.Component.TypeConfig
-> extends Button<TemplateSpec, TypeConfig> {}
+type ButtonSmallStyle = ButtonStyle & {
+  minWidth: number;
+  paddingX: number;
+  prefixH: number;
+  suffixH: number;
+  textStyle: TextBoxStyle;
+};
+
+declare namespace ButtonSmall {
+  export type TemplateSpec = Button.TemplateSpec;
+}
+declare class ButtonSmall<
+  TemplateSpec extends Button.TemplateSpec = Button.TemplateSpec,
```

Comment:
These should be `ButtonSmall.TemplateSpec`

Reply:
@ImCoolNowRight , Regarding your comment, I think that is correct because If there are no additional properties specific to ButtonSmall that need to be defined in its TemplateSpec so I think we can use Button.TemplateSpec, that way we are using the TemplateSpec structure defined in Button for ButtonSmall.

Reply:
as it is is correct but we shouldnt be declaring empty namespaces, per my comment above